docs/src/api/class-browsertype.md

```diff
@@ -290,6 +295,14 @@ Specify environment variables that will be visible to the browser. Defaults to `
 Slows down Playwright operations by the specified amount of milliseconds. Useful so that you can see what is going on.
 Defaults to 0.
 
+### option: BrowserType.launchPersistentContext.showUserInput
```

Comment:
Will I see these indicators on the screenshots? Does that mean all my screenshot tests will just fail? Let's hide them when screenshotting?

Reply:
@dgozman as a counter argument to this, what if I _want_ indicators in my screenshots? To make it more obvious what the screenshot is demonstrating?

Reply:
yeah, I would prefer to have them as well